storage-client/src/client.rs

```diff
@@ -115,6 +98,65 @@ impl StorageClient for StorageClientImpl {
         }
     }
 
+    async fn get_info_from_catalog_test(&self, _request: StorageRequest) -> Result<RequestParams> {
+        let bucket = "tests-parquet".to_string();
+        let keys = vec!["userdata1.parquet".to_string()];
+        Ok(RequestParams::S3((bucket, keys)))
```

Comment:
Is this test a mock to read a local file?

Reply:
Yes. And I think the return value should be `RequestParams::MockS3`. I'll update it.